debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/AbstractMysqlDefaultValueTest.java

```diff
@@ -317,4 +320,36 @@ public void parseNumericAndDecimalToDecimalDefaultValue() {
         assertThat(table.columnWithName("B").defaultValue()).isEqualTo(BigDecimal.valueOf(2.321));
         assertThat(table.columnWithName("C").defaultValue()).isEqualTo(BigDecimal.valueOf(12.678));
     }
+
+    @Test
+    public void parseTimeDefaultValue() {
+        String sql = "CREATE TABLE TIME_TABLE (\n" +
+                "  A timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,\n" +
+                "  B timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',\n" +
```

Comment:
Do we actually need the explicit line feeds? If so, let's have them for all lines consistently, although I don't think we need them at all.